be/src/runtime/runtime_state.h

```diff
@@ -77,6 +77,8 @@ namespace pipeline {
 class QueryContext;
 }
 
+constexpr int64_t kRpcHttpMinSize = ((1L << 31) - (1L << 20));
```

Comment:
why not use (1<<31) ? substract (1<<20) for what?

Reply:
avoid the case the serialized size may be larger than pre-computed.